src/applications/burials/helpers.jsx

```diff
@@ -163,18 +163,32 @@ export const transportationWarning = (
   </div>
 );
 
-export const burialDateWarning = (
-  <div className="usa-alert usa-alert-warning background-color-only">
-    <span>
-      If filing for a non-service-connected allowance, the Veteran’s burial date
-      must be no more than 2 years from the current date. Find out if you still
-      qualify.{' '}
-      <a href="/burials-memorials/eligibility/" target="_blank">
-        Learn about eligibility
-      </a>
-    </span>
-  </div>
-);
+export const BurialDateWarning = () => {
+  const [shouldRender, setShouldRender] = useState(false);
+
+  useEffect(() => {
+    // NOTE: If we don't wait at least 900ms to render,
+    // the alert content gets overridden by the year content.
+    // Using 1000ms to give a bit of padding
+    const timeout = setTimeout(() => setShouldRender(true), 1000);
+    return () => clearTimeout(timeout);
+  }, []);
+
+  return (
+    shouldRender && (
+      <va-alert aria-live="polite" background-only show-icon status="warning">
```

Comment:
@joshkimux Correct me if I'm wrong, please, but doesn't an `aria-live` element need to exist on the page before adding & removing content?

Reply:
Does it always have to exist on the page or is it enough for it to exist right before the element is added. I’m not 100% sure how to have the form system hide only part of the content of the ui:description

Reply:
Ideally we want it to exist on the page; it performs best that way. If we can't do that, setting a delay usually between the container `div` being added and the content being added will do the trick.